app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,99 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.dto.Task;
+import com.codesoom.assignment.generators.IdGenerator;
+import com.codesoom.assignment.repositories.TaskRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * 할 일에 대한 HTTP request를 처리합니다.
+ */
+@RestController
+@RequestMapping(path = "/tasks")
+@CrossOrigin
+public class TaskController {
+
+    private final TaskRepository taskRepository;
+    private final IdGenerator taskIdGenerator;
+
+    @Autowired
+    public TaskController(TaskRepository taskRepository, IdGenerator unsignedLongGenerator) {
+        this.taskRepository = taskRepository;
+        this.taskIdGenerator = unsignedLongGenerator;
+    }
+
+    /**
+     * 모든 할 일의 리스트를 반환한다.
+     *
+     * @return 할 일 목록
+     */
+    @GetMapping
+    public List<Task> getAllTasks() {
+        return this.taskRepository.findAll();
+    }
+
+    /**
+     * 새로운 할 일을 생성해 리턴합니다.
+     *
+     * @param task 생성할 할 일 정보
+     * @return 새로 생성된 할 일
+     */
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public Task createTask(@RequestBody Task task) {
+        task.setId(taskIdGenerator.generateId());
+        return this.taskRepository.addTask(task);
+    }
+
+    /**
+     * PathVariable로 요청된 id를 가진 할 일 인스턴스를 반환한다.
+     *
+     * @param id 유저가 얻기를 원하는 할 일의 id이다.
+     * @return 유저가 요청한 id를 가진 할 일 인스턴스를 반환한다.
+     */
```

Comment:
주석에서 굳이 `PathVariable`을 언급할 필요는 없어 보여요.
다음과 같이 해도 충분할 것 같습니다.
```suggestion
    /**
     * 요청된 id를 가진 할 일 인스턴스를 반환한다.
     *
     * @param id 조회 대상 할 일의 id
     * @return 조회된 할 일 인스턴스
     */
```

Reply:
주석에서 불필요할 것 같은 내용을 줄여나갈 수 있도록 주의하겠습니다. 그리고 `@param`과 `@return`은 간결하게 작성할 수 있도록 주의하겠습니다.